### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**

### CODE REVIEW

File: .idea/dictionaries/spine_examples.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="spine-examples">
+    <words>
+      <w>assignability</w>
+      <w>protobuf</w>
+      <w>suppressions</w>
+    </words>
+  </dictionary>
+</component>
```
Comment (armiol) [inline .idea/dictionaries/spine_examples.xml]: Newline is missing.

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="PROJECT_PROFILE" value="Project Default" />
+    <option name="USE_PROJECT_PROFILE" value="true" />
+    <version value="1.0" />
+  </settings>
+</component>
```
Comment (armiol) [inline .idea/inspectionProfiles/profiles_settings.xml]: Newline is missing.

File: server-examples/src/main/java/org/spine3/examples/aggregate/ClientApp.java [hunk 3]
```diff
@@ -89,10 +89,12 @@ public void onCompleted() {
...
 
-    /** Construct the client connecting to server at {@code host:port}. */
-    public ClientApp(String host, int port) {
+    /**
+     * Construct the client connecting to server at {@code host:port}.
+     */
+    private ClientApp(String host, int port) {
         commandFactory = CommandFactory.newBuilder()
-                            .setActor(newUserId(Identifiers.newUuid()))
+                            .setActor(UserId.newBuilder().setValue(Identifiers.newUuid()).build())
```
Comment (armiol) [inline server-examples/src/main/java/org/spine3/examples/aggregate/ClientApp.java]: Can we please extract the `userId` variable to enable auto-watch of its value during debugging?

**Comment by armiol** [comment]
@alexander-yevsyukov please see my comments.

**Comment by alexander-yevsyukov** [comment]
@alexander-aleksandrov, please pick this PR as a low priority task.

**Comment by armiol** [comment]
@alexander-yevsyukov closing this PR, as the same is being addressed in #6.

**Pull request closed without merging**